### PR TITLE
chore: add `setRemoveElement` and tests

### PR DESCRIPTION
**Pull request by poppoerika**
Added `setRemoveElement` since this operation is needed to implement `flush` for `MomentoTaggedCache` (https://github.com/momentohq/laravel-cache/issues/9).
Also, added tests for this operation.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            "cwd": "${workspaceRoot}",
+            "port": 9003,
+            "serverReadyAction": {
+                "pattern": "Development Server \\(http://localhost:([0-9]+)\\) started",
+                "uriFormat": "http://localhost:%s",
+                "action": "openExternally"
+            }
+        }
+    ]
+}
```
Comment (cprice404) [inline .vscode/launch.json:48]: i'm not sure if this file should be checked in?

**Review by cprice404 — APPROVED** [review]

**Comment by cprice404** [comment]
lgtm other than I'm not sure if we want that vscode file committed to vcs

File: .vscode/launch.json (hunk 1 above)
Comment (poppoerika) [reply .vscode/launch.json:48]: 😮‍💨

Comment (poppoerika) [reply .vscode/launch.json]: Will remove that.

**Review by cprice404 — APPROVED** [review]

**Pull request merged**